src/components/fullscreen/style.less

```diff
@@ -48,9 +72,32 @@
 // }
 
 :host-context(.app) {
-  ::content {
-    .c-fullscreen {
-      margin-right:-7px;
+
+  a {
+    padding-top: 6px;
+    padding-bottom: 0;
+
+    .icon-fullscreen {
+      font-size: 1em;
+      margin-right:-8px;
+
+      &:before {
+        padding: 2px 5px;
+        border-radius: 3px;     
+      }
+    }
+  }
+}
+
+:host-context(.app.header-is-sticky) {
+
+  a {
+
+    .icon-fullscreen {
```

Comment:
This rule will be applied to all fullscreen icon, anywhere you place it. Is this really intended?

Reply:
Noop. Will look into it.